### DISCUSSION TITLE
Is ScriptKit supposed to be a Command Palette like Raycast or Alfred?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [johnlindquist/kit#1413] [General]
That was the impression I got when I first found it the other day, but I'm noticing that the described purpose of this app is different from and doesn't have as many command palette features as those two. It would be nice to have a free or more free alternative to the two. Alfred is pretty expensive, while Raycast has a pro subscription and is VC backed (who knows where that could lead Raycast?!).

**Comment by johnlindquist** [comment]
@YousufSSyed Which "command palette features" do you think are missing?
